src/components/clusterConfiguration/ClusterConfigurationForm.tsx

```diff
@@ -222,84 +136,15 @@ const ClusterConfigurationForm: React.FC<ClusterConfigurationFormProps> = ({
-                >
-                  Install Cluster
-                </ToolbarButton>
-                <ToolbarButton
-                  type="submit"
-                  name="save"
-                  variant={ButtonVariant.secondary}
-                  isDisabled={isSubmitting || !isValid || !dirty}
-                  onClick={submitForm}
-                  id="cluster-configuration-save"
-                >
-                  Validate & Save Changes
-                </ToolbarButton>
-                <ToolbarButton
-                  variant={ButtonVariant.secondary}
-                  isDisabled={isSubmitting || !dirty}
-                  onClick={() => resetForm()}
-                  id="cluster-configuration-discard"
-                >
-                  Discard Changes
-                </ToolbarButton>
-                {isSubmitting && (
-                  <ToolbarText component={TextVariants.small}>
-                    <Spinner size="sm" id="cluster-configuration-spinner-saving-changes" /> Saving
-                    changes...
-                  </ToolbarText>
-                )}
-                {isStartingInstallation ? (
-                  <ToolbarText component={TextVariants.small}>
-                    <Spinner size="sm" id="cluster-configuration-spinner-starting-installation" />{' '}
-                    Starting installation...
-                  </ToolbarText>
-                ) : (
-                  <ToolbarText component={TextVariants.small}>
-                    {!Object.keys(errors).length && !dirty && cluster.status === 'ready' ? (
-                      <>
-                        <CheckCircleIcon
-                          color={successColor.value}
-                          id="cluster-configuration-icon-ready-to-install"
-                        />{' '}
-                        The cluster is ready to be installed.
-                      </>
-                    ) : (
-                      <>
-                        <Button
-                          variant={ButtonVariant.link}
-                          onClick={() => setIsValidationSectionOpen(!isValidationSectionOpen)}
-                          id="cluster-configuration-button-not-ready-to-install"
-                          isInline
-                        >
-                          <WarningTriangleIcon color={warningColor.value} />{' '}
-                          <small>The cluster is not ready to be installed yet</small>
-                        </Button>
-                      </>
-                    )}
-                  </ToolbarText>
-                )}
-              </ClusterToolbar>
+              <ClusterWizardToolbar
+                cluster={cluster}
+                errors={errors}
+                dirty={dirty}
+                isSubmitting={isSubmitting}
+                isNextDisabled={!canNextClusterConfiguration({ isValid, isSubmitting, cluster })}
+                onNext={submitForm}
+                onBack={onBack}
```

Comment:
As described in other comments, I'd prefer to just use `() => setCurrentStepId('networking')`